events_structs.go

```diff
@@ -755,6 +803,71 @@ func (ev *EventDeleteNotifierType) String() string {
 	return fmt.Sprintf("Notifier '%s' deleted on %s app", d.NotifierName, ev.AppName)
 }
 
+// Security events
```

Comment:
We started splitting to multiple files the events. Maybe add all these security related events in a file `events_security.go`?